apache-rat-core/src/test/java/org/apache/rat/OptionCollectionTest.java

```diff
@@ -118,27 +121,44 @@ public void testDeprecatedUseLogged() throws IOException {
         TestingLog log = new TestingLog();
         try {
             DefaultLog.setInstance(log);
-            String[] args = {longOpt(OptionCollection.DIR), "foo", "-a"};
-            ReportConfiguration config = OptionCollection.parseCommands(args, (o) -> {
-            }, true);
-
+            String[] args = {longOpt(OptionCollection.DIR), "target", "-a"};
+            ReportConfiguration config = OptionCollection.parseCommands(args, o -> fail("Help printed"), true);
         } finally {
             DefaultLog.setInstance(null);
         }
         log.assertContains("WARN: Option [-d, --dir] used.  Deprecated for removal since 0.17: Use '--'");
+        log.assertNotContains("WARN: Option [-d, --dir] used.  Deprecated for removal since 0.17: Use '--'", 1);
         log.assertContains("WARN: Option [-a] used.  Deprecated for removal since 0.17: Use '-A' or '--addLicense'");
+        log.assertNotContains("WARN: Option [-a] used.  Deprecated for removal since 0.17: Use '-A' or '--addLicense'", 1);
+    }
+
+    @Test
+    public void testDirOptionCapturesDirectoryToScan() throws IOException {
+        TestingLog log = new TestingLog();
+        ReportConfiguration config = null;
+        try {
+            DefaultLog.setInstance(log);
+            String[] args = {longOpt(OptionCollection.DIR), "foo"};
+            config = OptionCollection.parseCommands(args, (o) -> {
+            }, true);
+        } finally {
+            DefaultLog.setInstance(null);
+        }
+        assertThat(config).isNotNull();
+        log.assertContains("WARN: Option [-d, --dir] used.  Deprecated for removal since 0.17: Use '--'");
+        log.assertNotContains("WARN: Option [-d, --dir] used.  Deprecated for removal since 0.17: Use '--'", 1);
```

Comment:
Is the semantics of both lines supposed to be doesNotContainStringMoreThanExactlyOnce? Rading it one by one seems a bit confusing to me ..... sry

Reply:
Should we merge both assertions into one or only use exactly one?